targets/TARGET_STM/rtc_api.c

```diff
@@ -358,17 +356,18 @@ uint32_t rtc_read_lp(void)
     timeinfo.tm_min  = RTC_Bcd2ToByte((uint8_t)((Read_time & (RTC_TR_MNT | RTC_TR_MNU)) >> 8));
     timeinfo.tm_sec  = RTC_Bcd2ToByte((uint8_t)((Read_time & (RTC_TR_ST  | RTC_TR_SU))  >> 0));
 
-    uint32_t RTC_time_s = timeinfo.tm_sec + timeinfo.tm_min * 60 + timeinfo.tm_hour * 60 * 60; // Max 0x0001-517F => * 8191 + 8191 = 0x2A2E-AE80
+    uint32_t RTC_time_s = (timeinfo.tm_sec + timeinfo.tm_min * 60 + timeinfo.tm_hour * 60 * 60) * PREDIV_S_VALUE + PREDIV_S_VALUE - Read_SubSeconds; // Max 0x0001-517F * 8191 + 8191 = 0x2A2E-AE80
 
     if (LP_last_RTC_time <= RTC_time_s) {
         LP_continuous_time += (RTC_time_s - LP_last_RTC_time);
```

Comment:
Would be great to explain in commit message how those 3 variables are used:
LP_continuous_time : is it the time in µsec since start-up ? 
RTC_time_s: the current time that was just read from RTC register ?
LP_last_RTC_time: the last time read from RTC register, either in this function, or in RTC_write()  ?)
I don't undertstand the "add 24h" case  ...